plugins/virtual_event/services/VirtualEventService.php

```diff
@@ -178,16 +183,23 @@ protected function getGroup($groupId)
 		}
 	}
 	
-	protected function getScheduleEvent($eventId, $subType)
+	/**
+	 * Can be used from derived classes to set additional filter that don't automatically happen in applyPartnerFilters
+	 *
+	 * @param string $peer
+	 */
+	protected function applyPartnerFilterForClass($peer)
```

Comment:
you already have this function in your parent class kalturaBaseService, no need to copy